src/components/item/sharing/ItemMembershipSelect.tsx

```diff
@@ -71,9 +56,9 @@ const ItemMembershipSelect = ({
       values={Object.values(PermissionLevel).map((v) => ({
         value: v,
         text: enumT(v),
-        disabled: disabledMap[v],
+        disabled,
       }))}
-      disabled={typeof disabled === 'boolean' ? disabled : undefined}
+      disabled={disabled}
```

Comment:
This is ignored by the current version of UI. See https://github.com/graasp/graasp-ui/issues/646